pkg/repository/prisma/dbsqlc/events.sql

```diff
@@ -82,6 +82,32 @@ INSERT INTO "Event" (
     @additionalMetadata::jsonb
 ) RETURNING *;
 
+-- name: CreateEvents :copyfrom
+INSERT INTO "Event" (
+    "id",
+    "key",
+    "tenantId",
+    "replayedFromId",
+    "data",
+    "additionalMetadata"
+
+) VALUES (
+    $1,
+    $2,
+    $3,
+    $4,
+    $5,
+    $6
+);
+
+
+-- name: GetInsertedEvents :many
+
+SELECT * FROM "Event"
+WHERE xmin::text = (txid_current() % (2^32)::bigint)::text
```

Comment:
why are we using `xmin` here?

Reply:
This is how we are retrieving the inserted events. Copyfrom doesn't return anything and so in order to find out what events where changed we are getting the transaction id and finding out what events were last modified using our current transaction id. Also why I needed to wrap in a txn. 

We need to retrieve the events so we can call the callbacks on them once inserted.